packages/client/hmi-client/src/components/model/petrinet/tera-model-equation.vue

```diff
@@ -9,17 +9,22 @@
 		@update-model-from-equation="updateModelFromEquations"
 	>
 		<template #math-editor>
-			<tera-math-editor
-				v-for="(eq, index) in equations"
-				:key="index"
-				:index="index"
-				:is-editable="isEditable"
-				:is-editing-eq="isEditing"
-				:latex-equation="eq"
-				@equation-updated="setNewEquation"
-				@delete="deleteEquation"
-				ref="equationsRef"
-			/>
+			<div v-if="exceedEquationsThreshold == false">
+				<tera-math-editor
+					v-for="(eq, index) in equations"
+					:key="index"
+					:index="index"
+					:is-editable="isEditable"
+					:is-editing-eq="isEditing"
+					:latex-equation="eq"
+					@equation-updated="setNewEquation"
+					@delete="deleteEquation"
+					ref="equationsRef"
+				/>
+			</div>
+			<div v-else>
+				<p>Number of equations exceed LaTex generator threshold.</p>
+			</div>
```

Comment:
No need for superflous elements

```suggestion
			<template v-if="exceedEquationsThreshold == false">
				<tera-math-editor
					v-for="(eq, index) in equations"
					:key="index"
					:index="index"
					:is-editable="isEditable"
					:is-editing-eq="isEditing"
					:latex-equation="eq"
					@equation-updated="setNewEquation"
					@delete="deleteEquation"
					ref="equationsRef"
				/>
			</template>
			<p v-else>Number of equations exceed LaTex generator threshold.</p>
```

Reply:
Linter doesn't like this as it thinks the `v-if` does not exist